mailarchive/mailarchive-subetha/src/java/org/sakaiproject/mailarchive/SakaiMessageHandlerFactory.java

```diff
@@ -185,7 +187,25 @@ public void recipient(String to) throws RejectException {
                 if (StringUtils.isNotBlank(fromReply) && to.startsWith(serverConfigurationService.getString(FROM_REPLY, StringUtils.EMPTY))) {
                     isMessageId = true;
                     String id = to.replace(serverConfigurationService.getString(FROM_REPLY), StringUtils.EMPTY).split("@")[0];
-                    currentMessage = synopticMsgcntrManager.getPvtMessageManager().getPrivateMessage(id);
+                    try {
+                        currentMessage = synopticMsgcntrManager.getPvtMessageManager().getPrivateMessage(id);
+                    } catch (MessagingException me) {
+                        if (me.getMessage().startsWith(MESSAGE_ERROR_521)) {
+                            // BOUNCE REPLY - send a message back to the user to let them know their email failed
+                            String errMsg = rb.getString("mail.support.521") + "\n\n";
+                            String mailSupport = StringUtils.trimToNull(serverConfigurationService.getString("mail.support"));
```

Comment:
Please move this mailSupport variable to line 193 ...... and then it can be removed from line 203